src/Routes/Books/Books.jsx

```diff
@@ -24,9 +24,9 @@ const Books = () => {
 
   return (
     <>
-      <div className="container border rounded my-3 p-3">
-        { Object.keys(books).map((key) => (
-          <Book key={key} idBook={key} book={books[key][0]} />
+      <div className="container my-3">
+        { Object.keys(books).map((key, index) => (
+          <Book key={key} idBook={key} book={books[key][0]} index={index} />
```

Comment:
- [ ] As required by the previous reviewer, 
 Kindly match the font on the left part of the design. The section listing the links should be blue in color as we have on the design:
 
 
<img width="287" alt="image" src="https://user-images.githubusercontent.com/52934128/222935526-823ded6a-a720-49fd-8e02-f8c97d45ef66.png">

Reply:
Thanks for the information, I am changing the style to match with template